pekko-sample-persistence-dc-scala/src/main/resources/application.conf

```diff
@@ -52,8 +52,8 @@ akka.persistence.cassandra {
 datastax-java-driver.advanced.reconnect-on-init = true
 
 
-# Akka Management config: https://developer.lightbend.com/docs/akka-management/current/index.html
-akka.management {
+# Apache Pekko Management config: https://developer.lightbend.com/docs/akka-management/current/index.html
```

Comment:
Note: These docs are not being produced yet so I haven't updated the URL

Reply:
sure - leave it for now then